File: jambeth/jambeth-merge/src/main/java/de/osthus/ambeth/merge/IMergeServiceExtensionExtendable.java
package de.osthus.ambeth.merge;

public interface IMergeServiceExtensionExtendable
{
	void registerMergeServiceExtension(IMergeServiceExtension mergeServiceExtension, Class<?> entityType);

	void unregisterMergeServiceExtension(IMergeServiceExtension mergeServiceExtension, Class<?> entityType);
}

File: ambeth/Ambeth.Merge/ambeth/merge/IMergeServiceExtensionExtendable.cs
using System;

namespace De.Osthus.Ambeth.Merge
{
    public interface IMergeServiceExtensionExtendable
    {
        void RegisterMergeServiceExtension(IMergeServiceExtension mergeServiceExtension, Type entityType);

        void UnregisterMergeServiceExtension(IMergeServiceExtension mergeServiceExtension, Type entityType);
    }
}